shared/actions/search.js

```diff
@@ -13,6 +12,8 @@ import type {ExtraInfo, Search, Results, SelectPlatform, SelectUserForInfo,
 import type {apiserverGetRpc, apiserverGetResult} from '../constants/types/flow-types'
 import type {TypedAsyncAction} from '../constants/types/flux'
 
+const {platformToLogo16, platformToLogo32} = Constants
```

Comment:
you can't mix import * as Constants and {a,b,c} so we have to import it and extract it